packages/portal-proto/src/components/Modals/SetModals/SaveSelectionModal.tsx

```diff
@@ -127,6 +132,8 @@ const SaveSelectionAsSetModal: React.FC<SaveSelectionAsSetModalProps> = ({
               filters: filters ?? {},
               size: form.values.top,
               score: sort,
+              set_type: "mutable",
+              intent: "user",
             })
```

Comment:
<img width="1225" alt="Screenshot 2024-04-05 at 2 56 04 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/387c626b-4ac8-40a8-82ef-393a5ed6fd61">

Reply:
ephemeral -- we don't guarantee these things stick around forever (just like in v1)